code/style.css

```diff
@@ -4,6 +4,92 @@ Feel free to change or remove this styling!
 We want you focus A LOT on the styling for this project. 
 */
 
+@import url('https://fonts.googleapis.com/css2?family=Khand:wght@300&family=Parisienne&display=swap');
+
 body {
-  background-color: peachpuff;
+  background-color: lightgray;
+  color: black;
+
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+fieldset {
+  padding: 20px;
+  border-radius: 20px;
+  border: 7px dotted #f7c163;
+}
+
+legend {
+  font-size: 30px;
+  text-align: center;
+  font-family: 'Parisienne', cursive;
+  text-shadow: 0.1em 0.1em 0.2em black
+}
+
+label {
+  font-family: 'Khand', sans-serif;
+}
+
+p {
+  font-family: 'Khand', sans-serif;
+}
+
+.sender {
+  display: flex;
+  flex-direction: column;
+  justify-content: stretch;
+}
+
+.wrapper-col {
+  display: flex;
+  flex-direction: column;
 }
+
+.receiver {
+  display: flex;
+  flex-direction: column;
+  justify-content: stretch;
+  margin-top: 15px;
+}
+
+button {
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  justify-items: center;
+  margin-top: 30px;
+  font-family: 'Khand', sans-serif;
+  font-size: 15px;
+  background-color: lightgray;
+  border-color: #f7c163;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+button:hover {
+  background-color: grey;
+}
+
+textarea {
```

Comment:
You can add `resize: none;` if you don't want the user to be able to resize the textarea :)